LICENSE

```diff
@@ -1,3 +1,6 @@
+Contains information from the language-subtag-registry JSON Database (https://github.com/mattcg/language-subtag-registry/tree/master/data/json)
```

Comment:
Can you remove this blurb here https://github.com/yext/chat-headless/pull/25/files#diff-c693279643b8cd5d248172d9c22cb7cf4ed163a3c98c8a3f69c2717edd3eacb7R4? it's the same as the one you have above, except that it remove a newline. This cause the script from one of the deps to not detect that we already include the blurb and adds it again.

Reply:
sure thing!